Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM iojs:1.6
+FROM node:4
```

Comment:
No then it's probably fine, just wanted to make sure i could still run tests for all dialects including pg-native :)